DEPLOYMENT-CHECKLIST.md

```diff
@@ -0,0 +1,71 @@
+# Deployment Checklist
+
+Use this checklist to validate a successful Guardian Connector deployment before handoff.  
+It covers the manual steps that are not already automated through the scripts and workflows in this repo, and is meant to complement — not replace — the deployment guides.
+
+## VM
+
+- [ ] Did I set a `*.alias` A record in my DNS provider? (And can confirm it resolves to the VM's IP?)
+- [ ] Did I set an appropriate file quota (e.g. 1TB) for file shares?
```

Comment:
if you merge before #53, I'll update #53 to remove this line.

Reply:
Merging!